README.md

```diff
@@ -24,11 +32,8 @@ To retrieve the token run the following command:
 sudo kubectl get secret admin-user -n kubernetes-dashboard -o jsonpath={".data.token"} | base64 -d
 ```
 
-The Dashboard can be reached through a **NodePort service**. The ```dashboard``` role handles the configuration of a NodePort service.
-To retrieve the NodePort run the following command:
-```bash
-sudo kubectl --namespace kubernetes-dashboard get svc kubernetes-dashboard -o=jsonpath="{.spec.ports[0].nodePort}"
-```
+The dashboard is reachable via both HTTP and HTTPS at the address __*http(s)://{node IP address}/dashboard*__ thanks to the **LoadBalancer service**.
```

Comment:
```suggestion
The dashboard is reachable via both HTTP and HTTPS at the address __*http(s)://{node IP address}/dashboard*__ thanks to the **LoadBalancer service** enabled by default during K3s installation.
```

Reply:
The dashboard is not enabled during K3s installation, but there is another playbook for this.